mypyc/test-data/irbuild-basic.test

```diff
@@ -3646,3 +3646,69 @@ L0:
     r2 = r1 >= 0 :: signed
     r3 = truncate r1: int32 to builtins.bool
     return r3
+
+[case testRangeObject]
+def range_object() -> None:
+    r = range(4, 12, 2)
+    sum = 0
+    for i in r:
+        sum += i
+
+def range_in_loop() -> None:
```

Comment:
There probably is an existing test case for range in a for loop, so this can be removed.

Reply:
I cannot find the irbuild case and I'll keep this.